docs/VLSI/Basic-Flow.rst

```diff
@@ -0,0 +1,217 @@
+.. code-block:: shell
+
+    make buildfile CONFIG=GemminiRocketConfig VLSI_TOP=Gemmini tech_name=tsmintel3 INPUT_CONFS="example-design.yml example-tools.yml example-tech.yml"
+
+Running the VLSI Flow
+---------------------
+
+Running a basic VLSI flow using the Hammer default configurations is fairly simple, and consists of simpele ``Make`` command with the previously mentioned Make variables.
+
+Synthesis
+^^^^^^^^^
+
+In order to run synthesis, we run ``make syn`` with the matching Make variables. 
+Post-synthesis logs and collateral will be saved in ``build/<config-name>/syn-rundir``. The raw QoR data (area, timing, gate counts, etc.) will be found in ``build/<config-name>/syn-rundir/reports``.
+
+Hence, if we want to monolitically synthesize the entire SoC, the relevant command would be
+.. code-block:: shell
+
+    make syn CONFIG=<chipyard_config_name> tech_name=<tech_name> INPUT_CONFS="example-design.yml example-tools.yml example-tech.yml"
+
+In a more typical scenario of working on a single module, for example the Gemmini accelerator within the GemminiRocketConfig Chipyard SoC configuration, the relevant command would be
+.. code-block:: shell
+
+    make syn CONFIG=GemminiRocketConfig VLSI_TOP=Gemmini tech_name=tsmintel3 INPUT_CONFS="example-design.yml example-tools.yml example-tech.yml"
+
+
+It is worth checking the final-qor.rpt report to make sure that the synthesized design meets timing before moving to the place-and-route step.
+
+Place-and-Route
+^^^^^^^^^^^^^^^
+In order to run place-and-route, we run ``make par`` with the matching Make variables.
+Post-PnR logs and collateral will be saved in ``build/<config-name>/par-rundir``. Specifically, the resulting GDSII file will be in that directory with the suffix ``*.gds``. and timing reports can be found in ``build/<config-name>/par-rundir/timingReports``.
+Place-and-route is requires more design details in contrast to synthesis. For example, place-and-route requires some basic floorplanning constraints. The default ``example-design.yml`` configuration file template allows the tool (specifically, the Cadence Innovus tool) to use it's automatic floorplanning capability within the top level of the design (``ChipTop``). However, if we choose to place-and-route a specific block which is not the SoC top level, we need to change the top-level path name to match the ``VLSI_TOP`` make parameter we are using.
+
+Hence, if we want to monolitically place-and-route the entire SoC with the default tech plug-in parameters for power-straps and corners, the relevant command would be
+.. code-block:: shell
+
+    make par CONFIG=<chipyard_config_name> tech_name=<tech_name> INPUT_CONFS="example-design.yml example-tools.yml example-tech.yml"
+
+In a more typical scenario of working on a single module, for example the Gemmini accelerator within the GemminiRocketConfig Chipyard SoC configuration,
+
+.. code-block:: shell
+  vlsi.inputs.placement_constraints:
```

Comment:
I think you should add a sentence about how the width and height here could vary a lot based on the block you are building (is 300x300 even right for gemmini? I thought it was bigger).

Also this is a hammer comment but I think we might be able to support a different mode for this that is even more automatic. But that would need some manual reading, so its probably best just to be noted as an issue (in hammer).